### PR TITLE
renders review turnaround process

### PR DESCRIPTION
**Pull request by hvilloria**
## What does this PR do?

* renders review turnaround metric for a given project.

### CODE REVIEW

File: app/services/queries/daily_metrics.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      end
+    end
+
+    def users_project
+      @users_project ||= Project.find(@project_id).users_projects
+    end
+
+    def format_data(metrics)
+      dates_and_values = Hash.new
+      metrics.each do |metric|
```
Comment (hdamico) [inline app/services/queries/daily_metrics.rb:24]: What about an `inject` here?
For example, `metrics.each.inject({})`.

File: app/views/projects/review_turnaround.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class='p-3'>
+  <%= line_chart  @metrics, ytitle: "Time as minutes", suffix: " minutes" %>
```
Comment (santiagovidal) [inline app/views/projects/review_turnaround.erb:2]: Can we make it to hours?

File: db/seeds.rb [hunk 3]
```diff
@@ -8,3 +8,17 @@
 if Rails.env.development?
   AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password')
 end
+
+project = FactoryBot.create(:project, name: 'rs-code-review-metrics')
```
Comment (santiagovidal) [inline db/seeds.rb:12]: We should run all this only in `development`.

File: app/services/queries/daily_metrics.rb (hunk 1 above)
Comment (hvilloria) [reply app/services/queries/daily_metrics.rb:24]: much better!

Comment (horacio) [reply app/services/queries/daily_metrics.rb:24]: Other option: `metrics.each_with_object({}) { |metric, hash| ... }`

File: app/views/projects/review_turnaround.erb (hunk 2 above)
Comment (santiagovidal) [reply app/views/projects/review_turnaround.erb:2]: For example if a review took 30 minutes, the graph should show 0.5 hours.

Comment (hvilloria) [reply app/views/projects/review_turnaround.erb:2]: got it! i will review it

Comment (santiagovidal) [reply app/views/projects/review_turnaround.erb:2]: You didn't change this 🤔

Comment (hvilloria) [reply app/views/projects/review_turnaround.erb:2]: sorry, miss click

**Pull request merged**